pkg/image/image_test.go

```diff
@@ -96,3 +97,24 @@ func TestImageAdditionalMetadata(t *testing.T) {
 		})
 	}
 }
+
+func TestImage_SquashedTree(t *testing.T) {
+	t.Run("zero layers", func(t *testing.T) {
+		i := Image{
+			Layers: []*Layer{},
+		}
+
+		defer func() {
+			if r := recover(); r != nil {
+				t.Errorf("panicked (and recovered) while computing squashed tree for image with zero layers: %v", r)
+			}
+		}()
+
+		// Asserting that this call doesn't panic (regression: https://github.com/anchore/stereoscope/issues/56)
+		result := i.SquashedTree()
+
+		if result == nil {
+			t.Error("expected an initialized, empty FileTree, but got a nil FileTree")
```

Comment:
(probably a nit), but is there such a thing like a `nil FileTree` ? I thought that this would be a (plain) `nil`
```suggestion
			t.Error("expected an initialized, empty FileTree, but got nil")
```

Reply:
@alfredodeza Good question — Actually, all instances of `nil` in Go _are associated_ with a specific type. So it actually is a "nil FileTree". (This is different from `null` in other languages)